tests/admin/test_view.py

```diff
@@ -1138,7 +1190,7 @@ def test_delete_not_linked(self):
     def test_delete_linked(self):
         self._login()
 
-        email = FAKER.email()
+        email = FAKER.email()  # pylint: disable=no-member
```

Comment:
@snowcrshd If there are really no better way  to make pylint understand faker (i.e. plugin), I'd suggest to improve it a little bit:

* Creating a dedicated pylintrc file in `tests` folder - copy from top-level pylintrc (better, reference it, but I haven't found a way to do that)
* Put additional suppressions into it:
    * no-member
    * that rule that requires putting `len(array)` into variable first
* Change pylint run commands in travis.yml, tox.ini and Makefile to use it - pylint does not consider pylintrc files in nested folders as overrides, so you'd have to use pylint --rcfile=tests/pylintrc)